_scripts/start-containers.sh

```diff
@@ -0,0 +1 @@
+docker compose up
```

Comment:
I have made an extra restart sh script which does down, prune and up so that I don't have to do all that every time I try to make a fresh start. I hope that helps.